gravitee-apim-console-webui/src/management/configuration/theme/theme.component.ts

```diff
@@ -1,24 +0,0 @@
-/*
- * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-const ThemeComponent: ng.IComponentOptions = {
-  bindings: {
-    views: '<',
-  },
-  controller: 'ThemeController',
```

Comment:
This is the theme of the portal ? 🤔

Reply:
It is 🙃 I have FP and have saved the portal related assets and have renamed them. It still builds and runs no error 😅